### PR TITLE
KAA-1160 Glossary

### PR DESCRIPTION
**Pull request by DimaMir999**

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

File: doc/Glossary/index.md [hunk 1]
```diff
@@ -6,74 +6,232 @@ nav: /:path/Glossary/
...
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.
```
Comment (ashvayka) [inline doc/Glossary/index.md:73]: no space between "to it. Endpoint send"

Comment (ashvayka) [inline doc/Glossary/index.md:73]: There are multiple Access Tokens in this system. Endpoint Access Token, User Access token. Need to be more specific.

**Comment by ashvayka** [comment]
spaces are missing in a lot of sentences. Please use spell-check before you submit this for review.

File: doc/Glossary/index.md (hunk 1 above)
Comment (ashvayka) [inline doc/Glossary/index.md:73]: No link to Administration UI guide.

Comment (ashvayka) [inline doc/Glossary/index.md:73]: There is no Bootstrap server in this system. Kaa-node is the only one. Kaa node consists of three main services.

**Comment by ashvayka** [comment]
This is not ready. Major comments:
1) Please wait until https://github.com/kaaproject/kaa/pull/730 is merged.
2) Use main definitions from this PR
3) Avoid definitions that are inaccurate. Let's leave this for responsible authors of corresponding pages.
4) Fix all typos, spelling and spaces.

**Comment by rasendubi** [comment]
#730 is merged now.

File: doc/Glossary/index.md [hunk 2]
```diff
@@ -6,74 +6,239 @@ nav: /:path/Glossary/
...
```
Comment (ashvayka) [inline doc/Glossary/index.md:73]: no space after "also,". Please review grammar once again.

Comment (ashvayka) [inline doc/Glossary/index.md:73]: no space after "to it.". Please review grammar once again.

Comment (ashvayka) [inline doc/Glossary/index.md:73]: This is not true. If you are unsure about certain terminology, write smth like this:
Security token that is used during endpoint attachment procedure. Refer to [Endpoint Ownership] for more information.

Comment (ashvayka) [inline doc/Glossary/index.md:73]: also,Admin

**Comment by Oldnickus** [comment]
@kaa-leeroy 
add to whitelist

File: doc/Glossary/index.md (hunk 1 above)
Comment (ashvayka) [inline doc/Glossary/index.md:73]: please remove "(also, Admin UI and web UI)" Only one name in glossary per subject.

Comment (ashvayka) [inline doc/Glossary/index.md:73]: This is definitely not an online tool. This is a system component, a web interface, but not an online tool. Example of online tool: http://bfy.tw/6X0t

Comment (ashvayka) [inline doc/Glossary/index.md:73]: What is this TODO about?

**Comment by akokhanovskyi** [comment]
@hunch7 Can you please help finalize this section?

**Comment by hunch7** [comment]
Ok, got it.

**Comment by balldir** [comment]
I think this pr is deprecated and separat pr was opened for a new version.

**Comment by rasendubi** [comment]
Closing it then.

@DimaMir999 feel free to reopen if it's not deprecated.

**Pull request closed without merging**